docs/conf.py

```diff
@@ -240,16 +240,16 @@ def _get_rst_filepath_from_path(filepath: str):
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-if PACKAGE_NAME == 'apache-airflow':
-    html_static_path = ['apache-airflow/static']
+if PACKAGE_NAME in ['apache-airflow', 'helm-chart']:
+    html_static_path = [f'{PACKAGE_NAME}/static']
```

Comment:
Should we have a `common` or `shared` instead of duplicating the script? Maybe in a follow up?

Reply:
+1
These scripts look like they are almost entirely duplicates? I may be misunderstanding something.

Reply:
Yup, currently all our docs are isolated -- but we could do something with shared/common pieces